cobbler/api.py

```diff
@@ -243,7 +243,8 @@ def version(self, extended: bool = False) -> Union[float, Dict[str, Union[str, l
         if not extended:
             # for backwards compatibility and use with koan's comparisons
             elems = data["version_tuple"]
-            return int(elems[0]) + 0.1 * int(elems[1]) + 0.001 * int(elems[2])
+            # This double conversion is required because of the typical floating point problems.
+            return float(format(int(elems[0]) + 0.1 * int(elems[1]) + 0.001 * int(elems[2]), '.3f'))
```

Comment:
Do you think it is worth linking to the docs page you / Cédric got this from?

Reply:
Hm. It would be Python Standard Documentation. I am unsure if it is worth linking that. But after all I didn't find this myself, so yes I guess it would be better. Let me amend it.